[2025-01-03T07:13:51.602+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: insert_data.ft_balance_f scheduled__2024-12-31T00:00:00+00:00 [queued]>
[2025-01-03T07:13:51.645+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: insert_data.ft_balance_f scheduled__2024-12-31T00:00:00+00:00 [queued]>
[2025-01-03T07:13:51.653+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2025-01-03T07:13:56.941+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): ft_balance_f> on 2024-12-31 00:00:00+00:00
[2025-01-03T07:13:58.956+0000] {standard_task_runner.py:60} INFO - Started process 118 to run task
[2025-01-03T07:13:58.959+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'insert_data', 'ft_balance_f', 'scheduled__2024-12-31T00:00:00+00:00', '--job-id', '186', '--raw', '--subdir', 'DAGS_FOLDER/insert_data.py', '--cfg-path', '/tmp/tmp1iybz3pk']
[2025-01-03T07:13:58.963+0000] {standard_task_runner.py:88} INFO - Job 186: Subtask ft_balance_f
[2025-01-03T07:16:09.710+0000] {task_command.py:423} INFO - Running <TaskInstance: insert_data.ft_balance_f scheduled__2024-12-31T00:00:00+00:00 [running]> on host 403e7aadd7ee
[2025-01-03T07:16:09.959+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='agushchin' AIRFLOW_CTX_DAG_ID='insert_data' AIRFLOW_CTX_TASK_ID='ft_balance_f' AIRFLOW_CTX_EXECUTION_DATE='2024-12-31T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-31T00:00:00+00:00'
[2025-01-03T07:16:10.012+0000] {base.py:83} INFO - Using connection ID 'postgres-db' for task execution.
[2025-01-03T07:16:23.266+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-01-03T07:16:25.311+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=insert_data, task_id=ft_balance_f, execution_date=20241231T000000, start_date=20250103T071351, end_date=20250103T071625
[2025-01-03T07:16:44.518+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-01-03T07:16:48.642+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
